src/internal/packager2/inspect.go

```diff
@@ -142,3 +145,178 @@ func PackageInspectManifests(ctx context.Context, pkgLayout *layout2.PackageLayo
 
 	return PackageInspectManifestResults{Resources: resources}, nil
 }
+
+type DevInspectManifestsOptions struct {
+	CreateSetVariables map[string]string
+	DeploySetVariables map[string]string
+	Flavor             string
+	KubeVersion        string
+}
+
+type DevInspectManifestResults struct {
+	Resources []Resource
+}
+
+// DevInspectManifests returns manifests and Helm chart manifests after templating
+func DevInspectManifests(ctx context.Context, packagePath string, opts DevInspectManifestsOptions) (DevInspectManifestResults, error) {
```

Comment:
I'm not 100% on the API here. `Dev...` feels like a `cmd` concerns not `packager2` concerns, and `Package...` [L42](https://github.com/zarf-dev/zarf/pull/3658/files#diff-f12af45cbf249f1786d50fc2924a79099e8e712de195520443fe8bfa1b60a53bR42) repeats the go pkg name in a way. Thoughts on introducing an `inspect/` subdirectory and pkg in `internal/packager2`?

Reply:
Good callout. ATM I don't see that strong of a reason to separate inspect from the rest of packager2, it share code / functionality to find images with something like `dev inspect manifests`, but `package inspect manifest` does share code with pull so I'm a bit weary to introduce dependency complexity.

Another alternative for the API concerns could be `InspectDefinitionManifests` and `InspectPackageManifests` rather than the current `PackageInspectManifests` and `DevInspectManifests`.